pkg/eve/runme.sh

```diff
@@ -157,9 +157,17 @@ do_installer_raw() {
   dump "$OUTPUT_IMG" installer.raw
 }
 
+# create_installer_iso creates the installer iso and leaves it as /output.iso
+# common base for other usages like do_installer_iso and do_installer_net
+create_installer_iso() {
+  mkdir -p /installer_root
+  unsquashfs -f -d /installer_root /bits/installer.img 1>&2
```

Comment:
Do we always run this script with root permissions? Maybe it's not applicable here, but I remember when I was trying to do something similar to our images locally on my host, I had a problem with repacking the device files due to the permission preservation issue.